README.md

```diff
@@ -14,6 +14,14 @@ Integrate chat completion into your business applications with SAP Cloud SDK for
 
 _Insert a short description what is required to get your project running..._
 
+# 📚 Documentation
+- [AI Core APIs](./docs/ai-core-APIs.md)
+## Pre-requisites for AI Core Deployment
+- Enable the AI Core service in BTP
+    - [More Info](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup)
+- Create a configuration in AI Core using the `/configuration` endpoint
+    - [Example Usage](./docs/ai-core-APIs.md#create-a-configuration)
+
```

Comment:
[pp] Consider linking the documentation to the usage of AI core client here, instead of the section above.